avocado/utils/asset.py

```diff
@@ -117,9 +125,15 @@ def fetch(self):
                         path = urlobj.path
                     log.debug('Looking for file on %s.' % path)
                     if self._check_file(path):
-                        os.symlink(path, self.asset_file)
+                        try:
+                            os.symlink(path, self.asset_file)
+                        except OSError as e:
+                            if e.errno == errno.EEXIST:
+                                os.remove(self.asset_file)
+                                os.symlink(path, self.asset_file)
```

Comment:
If you expect the link to exist, then the `if os.path.exists()` instead of try/catch would be IMO more fitting, but I can also live with this.